webui/react/src/pages/TrialDetails/TrialInfoBox.test.tsx

```diff
@@ -230,4 +231,25 @@ describe('Trial Info Box', () => {
       expect(await screen.findByText('Forever')).toBeVisible();
     });
   });
+
+  describe('Lineage card', () => {
+    it('should show Data input card with tge lineage link when pachyderm integration data is present', () => {
+      const mockExperimentWith = Object.assign(mockExperiment, {
+        config: { ...mockExperiment.config, integrations: { pachyderm: mockIntegrationData } },
+      });
+
+      setup(mockTrial1, mockExperimentWith);
+      waitFor(() => {
+        expect(screen.findByText('Data Input')).toBeVisible();
+        expect(screen.findByText(mockIntegrationData.dataset.repo)).toBeVisible();
+      });
+    });
+
+    it('should not show Data input card when pachyderm integration is missing', () => {
+      setup(mockTrial1, mockExperiment);
+      waitFor(() => {
+        expect(screen.findByText('Data Input')).not.toBeVisible();
+      });
```

Comment:
i think these test cases actually dont assert values. even if text is changed to whatever values, the test cases pass. feels like this needs to be fixed before merging this PR

Reply:
the test cases are false negative. can we fix this?

Reply:
> changed to use data-testid in order to abstract the text values from it

i cant find test cases that use `data-testid`. i think its missing